src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CSharp.CodeFixes/HttpContextCurrentCodeFixer.cs

```diff
@@ -173,7 +173,7 @@ private static void InitializeHttpContextHelperInStartup(DocumentEditor editor,
             }
 
             // Add Initialize call in Configure method
-            var configureMethod = documentRoot.GetMethodDeclaration("Configure", "IApplicationBuilder");
+            var configureMethod = documentRoot.GetMethodDeclaration<MethodDeclarationSyntax>("Configure", "IApplicationBuilder");
```

Comment:
won't let me comment on the other section... but i noticed that we're doing `.WithWhitespaceTriviaFrom(configureServicesBody.Statements.First());`

can we always assume the configureServicesBody has a first statement?

Reply:
Hm... I suppose not. We do check that the method *exists*, but there's an edge case where it exists but is empty I guess. I'll update the check to make sure the method is both present and not empty before doing this.